main.go

```diff
@@ -62,14 +62,17 @@ func main() {
 	awsConfig := aws.NewConfig().WithRegion(config.RDSConfig.Region)
 	awsSession := session.New(awsConfig)
 
-	iamsvc := iam.New(awsSession)
 	rdssvc := rds.New(awsSession)
-	dbInstance := awsrds.NewRDSDBInstance(config.RDSConfig.Region, iamsvc, rdssvc, logger)
+	stssvc := sts.New(awsSession)
+
+	dbInstance := awsrds.NewRDSDBInstance(config.RDSConfig.Region, rdssvc, stssvc, logger)
 
 	sqlProvider := sqlengine.NewProviderService(logger)
 
 	serviceBroker := rdsbroker.New(config.RDSConfig, dbInstance, sqlProvider, logger)
 
+	go serviceBroker.CheckAndRotateCredentials()
```

Comment:
I'm unsure about error handling here. If we care about errors we should something like this:
```
go func() {
  err := serviceBroker.CheckAndRotateCredentials()
  if err != nil {
     logger.Error(err)
  }
}
```

If we want to log errors inside CheckAndRotateCredentials let's do not return them.